packages/components/src/components/post-tabs/readme.md

```diff
@@ -47,6 +47,14 @@ Type: `Promise<void>`
 | `"tabs"`    | Slot for placing tab headers. Each tab header should be a <post-tab-header> element. |
 
 
+## Shadow Parts
+
+| Part        | Description |
+| ----------- | ----------- |
+| `"content"` |             |
+| `"tabs"`    |             |
```

Comment:
Would be nice to have a description here and maybe in a separate ticket an explanation on how to style web component parts.

Reply:
There you go: #3171